### PR TITLE
Error: Already Piped FIX

### PR DESCRIPTION
**Pull request by harrshasri**
Chore which i think fixes this issue also
https://github.com/libp2p/js-libp2p-switch/issues/235

### CODE REVIEW

**Comment by harrshasri** [comment]
@diasdavid Can you accept this fix

**Review by daviddias — APPROVED** [review]
Thanks @harrshasri :)

**Comment by harrshasri** [comment]
@diasdavid Do we need to update this package dependency in js-ipfs v28.2 to v28.3

**Comment by daviddias** [comment]
@harrshasri note that secio was published as a patch version. The error you are seeing (already piped) is being fixed by https://github.com/libp2p/js-libp2p-switch/pull/245

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "libp2p-switch",
-  "version": "0.37.3",
+  "version": "0.37.4",
```
Comment (daviddias) [inline package.json:3]: ah crap. Please do not update the version manually, our release process does it.

Comment (daviddias) [reply package.json:3]: will fix in master

Comment (harrshasri) [reply package.json:3]: Oh Okay. Sorry. I'm a noob.

Comment (daviddias) [reply package.json:3]: no worries at all. Shame on me for not reviewing it properly :)

**Pull request merged**